packages/fscomponents/src/components/serializable/SerializableButton.tsx

```diff
@@ -13,6 +13,7 @@ import { ButtonProps } from '../Button';
 import { border, palette as defaultPalette } from '../../styles/variables';
 import { style as S, stylesSize, stylesTextSize } from '../../styles/Button';
 import { extractHostStyles } from '../../lib/style';
+import { useNavigator } from '@brandingbrand/fsapp';
```

Comment:
I would like to keep the props sorted in the following order.
```
type imports

react / react-native / redux

lodash / other utils / @brandingbrand

local / internal modules.
```

So for this import I would like for it to be between lines 10 and 12 with new lines on either side.

```
} from 'react-native';

import { useNavigator } from '@brandingbrand/fsapp';

import { ButtonProps } from '../Button';
```